design/relaying-semantics.md

```diff
@@ -0,0 +1,44 @@
+# Gravity bridge relaying semantics
+
+This document is designed to assist developers in implementing alternate Gravity relayers. The two major components of the Orchestrator which interact with Ethereum. The Gravity bridge has been designed for increased efficiency, not for ease of use. This means there are many implicit requirements of these external binaries which this document does it's best to make explicit.
```

Comment:
can you mention what the orchestrator is, it may be beneficial to also avoid using this term within this section as the relayer is only part of the orchestrator.

Reply:
that's a good call. I'll spend some more time disambiguating it.